styles/components/_tabbedModal.scss

```diff
@@ -30,6 +30,10 @@
     }
   }
 
+  .tabContent {
```

Comment:
Should we really be putting this max on the global component? I guess the question is if the designs will be that all tabbed modals will have a similar width. If that's the case, then this is fine.

(thinking out loud... it's probably ok for now and if some designs present themselves where this doesn't work, I suppose we could adjust).